src/alfasim_sdk/default_tasks.py

```diff
@@ -0,0 +1,455 @@
+    dst = Path(dst)
+    hook_specs_file_path = _get_hook_specs_file_path()
+    hm = HookManGenerator(hook_spec_file_path=hook_specs_file_path)
+    from alfasim_sdk._internal.constants import EXTRAS_REQUIRED_VERSION_KEY
+    from alfasim_sdk._internal.alfasim_sdk_utils import (
+        get_current_version,
+        get_extras_default_required_version,
+    )
+
+    package_name_suffix = f"sdk-{get_current_version()}"
+    extras_defaults = {
+        EXTRAS_REQUIRED_VERSION_KEY: get_extras_default_required_version()
+    }
+    hm.generate_plugin_package(
+        package_name,
+        plugin_dir,
+        dst,
+        extras_defaults=extras_defaults,
+        package_name_suffix=package_name_suffix,
+    )
+
+
+@task(
+    help={
+        "package-name": "Name of the package.",
+        "dst": "A path to where the output package should be created.",
+    }
+)
+def package(ctx, package_name, dst=os.getcwd()):
+    """
+    Create a new `<package-name>.hmplugin` file containing all the necessary files.
+
+    This command will first call `invoke compile` command to generate the shared library, and after that, the plugin
+    package will be generated with all the content available from the directory assets and artifacts.
+
+    By default, the `package` command will assume that the plugin project is the current directory and the generated file
+    will be placed also in the current directory.
+
+    In order to change that, it's possible to use the options `plugin-dir` and `dst`
+    """
+    compile(ctx)
+    package_only(ctx, package_name=package_name, dst=dst)
+
+
+@task()
+def update(ctx):
+    """
+    Update plugin files automatically generated by ALFAsim-SDK.
+
+    It is important to update the automatically generated files from each plugin when
+    a new version of ALFAsim-SDK is released. Otherwise compilation problems can occur.
+
+    The files updated are: |br|
+    - ´<plugin_folder>/src/hook_specs.h´ |br|
+    """
+    hook_specs_file_path = _get_hook_specs_file_path()
+    hm = HookManGenerator(hook_spec_file_path=hook_specs_file_path)
+
+    # Set to the plugin root, where the tasks.py is located in the plugin structure
+    plugin_folder = Path(ctx.config._project_prefix)
+    plugin_id = plugin_folder.name
+    plugin_location = plugin_folder.parent
+    hm.generate_hook_specs_header(plugin_id, plugin_location)
+
+
+@task()
+def install_plugin_to_user_folder(ctx):
+    """Install a specific plugin to $HOME/.alfasim_plugins"""
+    from pathlib import Path
+
+    # Set to the plugin root, where the tasks.py is located in the plugin structure
+    plugin_folder = Path(ctx.config._project_prefix)
+    plugin_id = plugin_folder.name
+
+    try:
+        hm_plugin_path = next(plugin_folder.glob("*.hmplugin"))
+    except StopIteration:
+        print_message(
+            f"Could not find any '*.hmplugin' in {plugin_folder}.",
+            color=Fore.RED,
+            bright=True,
+        )
+        exit(1)
```

Comment:
I think it has been commented before, but although this works, I believe the right way to go about here is to `raise invoke.Exit(code=1)`:

https://docs.pyinvoke.org/en/stable/api/exceptions.html#invoke.exceptions.Exit